.codecov.yml

```diff
@@ -6,4 +6,4 @@ coverage:
     project:
       default:
         enabled: true
-        target: 95%
+        target: 100%
```

Comment:
@jayvdb Do we enable the repository in codecov?

Reply:
How can this be 100% when the tests are not being run?

Reply:
The test is added in another PR